listenbrainz/webserver/templates/profile/info.html

```diff
@@ -35,13 +35,21 @@ <h3> Reset Last.FM Import timestamp </h3>
 
     <h3>User token</h3>
     <p>
-      If you would like to use an external program to submit data to ListenBrainz you will need the following
-      user token:
+      If you would like to use an external program to submit data to ListenBrainz you will need the user token:
     </p>
 
     <div class="form-inline form-group">
-      <input class="form-control" id="auth-token" type="text" size="36" value="{{ user.auth_token }}" readonly>
+      <p><b>Copy User Token</b>
+      <input class="form-control" id="auth-token" type="hidden" size="36" value="{{ user.auth_token }}" readonly>
       <button class="btn btn-info btn-sm glyphicon glyphicon-copy" id="copy-token" title="Copy token to clipboard"></button>
+      </p>
+    </div>
+
+    <div class="divcontainer">
+      <p><b>User Token</b>
+        <span class="btn btn-info glyphicon glyphicon-dropdown glyphicon-chevron-down" data-toggle="collapse" data-target="#collapsecontent" style="width: 50px"></span>
+        <div class="collapse" id="collapsecontent">{{ user.auth_token }}</div>
+      </p>
```

Comment:
Instead of this, we can simply add an "eye" icon which just toggles the type of `#copy-token` field from `password` to `text`, to reveal the key. I like [this](https://codepen.io/prasanthmj/pen/KqqBVo) one.

Reply:
Ideally, we'd want to have copy button to copy token ( without the need to copy-paste manually) and an option to show/hide. I am not sure if the link serves that purpose?
A token is not a password though, showing it in that sense can be a bit off imo.

Reply:
The copy button is to stay there. I was saying that having a copy button even though nothing is visible there is bad UX.

What I am saying is that we can have 2 buttons
1) Show/Hide
2) Copy

The copy button should copy the user token in both the cases when its plain string or password like string. 

The user can toggle the show/hide button to effectively show or hide the user token.

The user should know "Ah there is a string that I am trying to copy".  Currently the copy button simply copies and its confusing which string its talking about when there is none adjacent string whether text/password like.

But this is just an opinion. Lets wait for @paramsingh to review this.

Reply:
Gotcha. Sounds good to me.

Reply:
I think what @shivam-kapila says makes sense. the UX in the screenshot is a bit non-obvious.